src/App.js

```diff
@@ -19,7 +19,7 @@ const App = () => {
     chrome.init();
     const unregister = insights.chrome.on('APP_NAVIGATION', (event) => {
```

Comment:
This is no longer required. Your history is now shared with the chrome shell app.